<!DOCTYPE html>
<!--[if IE 8]> <html lang="en" class="ie8"> <![endif]-->  
<!--[if IE 9]> <html lang="en" class="ie9"> <![endif]-->  
<!--[if !IE]><!--> <html lang="en"> <!--<![endif]-->  
<head>
    <title>TeamSeason 2018</title>
    <!-- Meta -->
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="">
    <meta name="author" content="">    
    <link rel="shortcut icon" href="favicon.ico">  
    <!---<link href='http://fonts.googleapis.com/css?family=Roboto:400,400italic,500,500italic,700,700italic,900,900italic,300italic,300' rel='stylesheet' type='text/css'> 
    <link href='http://fonts.googleapis.com/css?family=Roboto+Slab:400,700,300,100' rel='stylesheet' type='text/css'>--->
    <!-- Global CSS -->
    <link rel="stylesheet" href="assets/plugins/bootstrap/css/bootstrap.min.css">   
    <!-- Plugins CSS -->    
    <link rel="stylesheet" href="assets/plugins/font-awesome/css/font-awesome.css">
    <link rel="stylesheet" href="assets/plugins/flexslider/flexslider.css">
    <!-- Theme CSS -->
    <link id="theme-style" rel="stylesheet" href="assets/css/styles.css">
		<link rel="stylesheet" href="assets/css/design.css">
		<link rel="stylesheet" href="assets/css/index3.css">
    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
    <!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->
</head> 

<body class="features-page"> 

	<!--导航栏-->
   <div id="navall">
    <div id="nav">
    	<div class="navpart">
    <dl class="list_dl">
        <dt class="list_dt" style="border-top-left-radius:6px;border-top-right-radius:6px;">
            <span class="_after"></span>
            <a href="#background" style="color:white;text-decoration: none;">Results & Analysis</a> 
            <i class="list_dt_icon"></i>              
        </dt>
       <dd class="list_dd">
            <ul>
              <div class="floatCtro">
                <li class="list_li">Circularization</li>
                <li class="list_li">HRCA</li>
                <li class="list_li">Sequence results </li>
                <li class="list_li" style="border-bottom-left-radius:6px;border-bottom-right-radius:6px;">Retrieve information</li>
                </div>
            </ul>
        </dd>
    
        <dt class="list_dt">
            <span class="_after"></span>
             <a href="#goals" style="color:white;text-decoration: none;">Materials & Methods</a>
         <i class="list_dt_icon"></i>   
        </dt> 

         <dd class="list_dd">
            <ul>
              <div class="floatCtro">
                <li class="list_li" style="padding-top:7px;">Materials</li>
                <li class="list_li" style=""> Methods</li>
            </ul>
        </dd>

    </dl>
</div>

</div>
</div>

    <!-- ******HEADER****** --> 
    <header id="header" class="header navbar-fixed-top">  
        <div class="container">       
            <h1 class="logo">
               <img id="season" src="assets/images/season.png" href="index.html">
            </h1><!--//logo-->
            <nav class="main-nav navbar-right" role="navigation">
                <div class="navbar-header">
                    <button class="navbar-toggle" type="button" data-toggle="collapse" data-target="#navbar-collapse">
                        <span class="sr-only">Toggle navigation</span>
                        <span class="icon-bar"></span>
                        <span class="icon-bar"></span>
                        <span class="icon-bar"></span>
                    </button><!--//nav-toggle-->
                </div><!--//navbar-header-->
                <div id="navbar-collapse" class="navbar-collapse collapse">
                    <ul class="nav navbar-nav">
                       <li class="nav-item"><a href="index.html">Home</a></li>
                        <li class="nav-item"><a href="project.html">PROJECT</a></li>
                        <li class="nav-item"><a href="design.html">DESIGN</a></li>
                       <li class="nav-item"><a href="simulation.html">SIMULATION</a></li>                    
                        <li class="active nav-item"><a href="experiment.html">EXPERIMENT</a></li>
						<li class="nav-item"><a href="discussion.html">DISCUSSION</a></li>
						<li class="nav-item"><a href="team.html">TEAM</a></li>
                    </ul><!--//nav-->
                </div><!--//navabr-collapse-->
            </nav><!--//main-nav-->                     
        </div><!--//container-->
    </header><!--//header-->
    

    <!-- ******Video Section****** --> 
    <section class="features-video section section-on-bg">
         <div class="container text-center">          
            <h2 style="font-size: 55px;margin-top: -65px;text-align: center;"><strong>Experiment</strong></h2>
<!--//video-container--> 
        </div><!--//container-->
   <section style="margin-left: 21%;margin-right:21%;text-align:justify;">

    <h2><strong>---What we did：</strong></h2>
    <p>---After designing the DNA single strand sequence, we set annealing and hybridization temperatures using a PCR instrument to self connect the DNA single strand to form assembly components, which were then hybridized to form a hybrid structure. We also demonstrated the formation of the desired structure through some methods.</p>
   </section><br>
   
	<div id="same01">
    <section style="margin-left: 21%;margin-right:21%;text-align:justify;">
     <h1><strong>Results & Analysis</strong></h1>  <br>
     <p style="font-size: 23px; "><strong>&nbsp;&nbsp;Sequence information</strong></p>
     <table class="experiment" width="750" border="1" style="margin-bottom:10px;" >
  <tbody>
    <tr>
      <th class="tabex" scope="col" width="150">Content</th>
      <th class="tabex" scope="col" width="600">sequence</th>
    </tr>
    <tr>
      <td class="tabex">---strand 1</td>
      <td class="tabex" width="550" style="padding:9px; white-space:normal; word-break:break-all;text-align:left;">---GAGTCCGTACCATCACATATTCACTATCAACCTATCATTCATAAGTCACTCAGGCTGTACGGACTCGCGC</td>
    </tr>
    
    <tr>
      <td  class="tabex">---strand 2</td>
      <td class="tabex" width="550" style="padding:9px; white-space:normal; word-break:break-all;text-align:left;" >---GACAGAGTTGAGCCTGAGTGACTTATGAATGATAGGTTGATAGTGAATATGTGATGCAACTCTGTCCGCG</td>
    </tr>
   
    <tr>
      <td class="tabex">---</td>
      <td class="tabex" style="padding:9px; white-space:normal; word-break:break-all;text-align: center;">---</td>
    </tr>
    <tr>
      <td class="tabex">---</td>
      <td class="tabex" style="padding:9px; white-space:normal; word-break:break-all;text-align: center;">---</td>
    </tr>
  </tbody>
</table>

<p class="figure" style="text-align:center; font-size:18px;"> &nbsp;---*All DNA strands were synthesized and purified by Sangon Biotech.</p>

<br>
     <div>
     <ol>
     	<div id="float01">
         <li style="font-size: 25px; "><strong>---Cyclization and hybridization</strong></li><br>
         <p style="padding-left: 180px;"><img width=400px; src="assets/images/experiment/1.png"></p>
         <p class="figure" style="text-align:center;"><strong>Figure 1. </strong>图片说明缺失</p><br>

          <p>---Samples were analyzed on a 10% denaturing polyacrylamide gel under 300V, the denaturing polyacrylamide were run in 1×TBE at room temperature.</p>
  
     <p>---We can obtain a dumbbell structure for hybridization, with lanes 4 and 5 showing the formation of dumbbells, and lane 6 showing the results of dumbbell hybridization.</p></div>

     <div id="float02">
     <li style="font-size: 25px; "><strong>---Hybridization involving DNA loops containing nicks</strong></li><br>
         <p style="padding-left: 180px;"><img width="400" src="assets/images/experiment/2.png"></p>

         <p class="figure" style="text-align:center;"><strong>Figure 2. </strong>图片说明缺失</p><br>

         <p>---Samples were analyzed by 3% agarose gel electrophoresis at 120V, and agarose gel electrophoresis at 1 × TBE operates at room temperature.</p>
       
       <p>---When self connecting, adding a certain proportion of unphosphorylated DNA single strands can obtain dumbbells containing nicks. Since dumbbells containing nicks are easier to hybridize than ordinary dumbbells, adjusting the proportion of unphosphorylated DNA single strands can also change the success rate of hybridization, enabling serial hybridization of multiple dumbbells.</p>
       <br>
</div>

<div id="float03">
  <li style="font-size: 25px; "><strong>【缺失内容】Analyse sequence results </strong></li><br>
    <p>Through MinION sequencing and simple sequencing results data processing, we can get DNA sequences in a short time, which can be directly used to retrieve Chinese characters information.In the end, a total of 52 files containing sequence information were obtained in fasta format.</p>
    <p>Step 1:</p>
    <ul>
        <li style="list-style-type: disc;margin-left:40px;">Use Nanopack software for preliminary data analysis.</li>
        <li style="list-style-type: disc;margin-left:40px;">After NanoLyse processing, DNA_CS. fasta was taken as the reference gene.</li>
        <li style="list-style-type: disc;margin-left:40px;">After NanoFilt treatment, the reads with a mass less than 1 and a length less than 1000 bp were removed.</li>
        <li style="list-style-type: disc;margin-left:40px;">After NanoPlot treatment, the total number of sequences, the average length and the average quality of all sequences were obtained. As the figure 3 and figure 4 show:</li>
    </ul><br>
    <p style="margin-left:160px;"><img src="assets/images/experiment/3.png"></p>

     <p class="figure" style="text-align:center;"><strong>Figure 3. </strong>Summary statistics of all sequences.</p><br><br>


    <p style="margin-left:160px;"><img src="assets/images/experiment/4.png"></p>

         <p class="figure" style="text-align:center;"><strong>Figure 4. </strong>Read lengths vs Average read quality plot.</p><br>

    <p>Most of reading lengths is within 1000 while average read quality focus on 7-9, which consistent with statistical information.</p>
     <p>Step 2:</p>
    <div style="margin-left:50px;">
    <ul>
        <li style="list-style-type: disc;"> Use Canu software for data splicing.</li>
        <li style="list-style-type: disc;"> The selected parameter is genomesize = 8 k and error Rate = 0.500.</li>
        <li style="list-style-type: disc;">After data splicing, all useful sequence information are obtained.</li>
        <li style="list-style-type: disc;">A total of 4 useful sequences were obtained. In one of the sequences, we found the sequence consistent with single strand DNA five times.</li><br>
        </ul>
    <br>
</div>



    <div id="float04">
    <li style="font-size: 25px; "><strong>【内容缺失】Retrieve information by software we designed</strong> </li><br>
    <p>After the sequence information is entered into the software we designed, the software retrieves the information by obtaining the reverse complementary information of the sequence, filtering, sorting, and so on. As shown in figure 5.</p><br>
    <p><img width="760" src="assets/images/experiment/9.png"></p>

    <p class="figure" ><strong>Figure 5. </strong>recovered information.“住地球村;” is recovered successfully, which we stored in DNA.</p><br>

   
    <p>Although the results of the sequencing include some error sequences with insertion, deletion or mismatch, we find the sequences containing the information we stored. Our software is designed to deal with insertion and deletion partly, <a href="discussion.html#aspect" style="text-decoration: underline;">details can be find in discussion</a>. It confirms the validity of our idea.
</p>
    <br>
    </div>
     </ol> 
     </div>
    </section>
</div>

<div id="mainword">
   <div id="same02">  
    <h1><strong>Materials & Methods</strong></h1>
    <br>
    <div id="float05" class="cur">

    <h3 style="font-size:30px;font-weight: bolder;">Materials</h3>
    <table class="experiment" width="750" border="1">
    <caption align="left" style=" caption-side:top;">
         <strong> 1. 10 x TBE Buffer</strong>
      </caption>
  <tbody>
    <tr>
      <th class="tabex" scope="col">Content</th>
      <th class="tabex" scope="col" width="375">Volume/Weight</th>
    </tr>
    <tr>
      <td class="tabex">tris</td>
      <td class="tabex" width="210">108 g</td>
    </tr>
    <tr>
      <td class="tabex">boric acid</td>
      <td class="tabex" width="210">55 g</td>
    </tr>
    <tr>
      <td class="tabex">disodium Ethylenediamine</td>
      <td class="tabex" width="210">7.44 g</td>
    </tr>
    <tr>
      <td class="tabex">water</td>
      <td class="tabex" width="210">add to 1000 ml</td>
    </tr>
    <tr>
      <td class="tabex">total</td>
      <td class="tabex" width="210">1 L</td>
    </tr>
  </tbody>
</table>
<br>
<table class="experiment" width="750" border="1">
    <caption align="left" style=" caption-side:top;">
        <strong> 2. 8 M urea acrylamide solution (20%)</strong>
      </caption>
  <tbody>
    <tr>
      <th class="tabex" scope="col">Content</th>
      <th class="tabex" scope="col" width="375">Volume/Weight</th>
    </tr>
    <tr>
      <td class="tabex">urea</td>
      <td class="tabex" width="210">192 g</td>
    </tr>
    <tr>
      <td class="tabex">acrylamide</td>
      <td class="tabex" width="210">77.3 g</td>
    </tr>
    <tr>
      <td class="tabex">bis-acrylamide</td>
      <td class="tabex" width="210">2.7 g</td>
    </tr>
    <tr>
      <td class="tabex">10 x TBE</td>
      <td class="tabex" width="210">40 ml</td>
    </tr>
    <tr>
      <td class="tabex">water</td>
      <td class="tabex" width="210">add to 400 ml</td>
    </tr>
    <tr>
      <td class="tabex">total</td>
      <td class="tabex" width="210">400 ml</td>
    </tr>
  </tbody>
</table>
<br>
<table class="experiment" width="750" border="1">
    <caption align="left" style=" caption-side:top;">
         <strong>3. Acrylamide solution (30%)</strong>
      </caption>
  <tbody>
    <tr>
      <th class="tabex" scope="col">Content</th>
      <th class="tabex" scope="col" width="375">Volume/Weight</th>
    </tr>
    <tr>
      <td class="tabex">acrylamide</td>
      <td class="tabex" width="210">116 g</td>
    </tr>
    <tr>
      <td class="tabex">bis-acrylamide</td>
      <td class="tabex" width="210">4 g</td>
    </tr>
    <tr>
      <td class="tabex">water</td>
      <td class="tabex" width="210">add to 400 ml</td>
    </tr>
    <tr>
      <td class="tabex">total</td>
      <td class="tabex" width="210">400 ml</td>
    </tr>
  </tbody>
</table>
<br>
<table class="experiment" width="750" border="1">
    <caption align="left" style=" caption-side:top;">
       <strong> 4. Denaturing polyacrylamide gels (10%)</strong>
      </caption>
  <tbody>
    <tr>
      <th class="tabex" scope="col">Content</th>
      <th class="tabex" scope="col" width="375">Volume/Weight</th>
    </tr>
    <tr>
      <td class="tabex">8 M urea acrylamide solution (20%)</td>
      <td class="tabex" width="210">20 ml</td>
    </tr>
    <tr>
      <td class="tabex">8 M ureasolution</td>
      <td class="tabex" width="210">20 ml</td>
    </tr>
    <tr>
      <td class="tabex">total</td>
      <td class="tabex" width="210">40 ml</td>
    </tr>
  </tbody>
</table>
<br>
<table class="experiment" width="750" border="1">
    <caption align="left" style=" caption-side:top;">
       <strong> 5. Native polyacrylamide gels (10%)</strong>
      </caption>
  <tbody>
    <tr>
      <th class="tabex" scope="col">Content</th>
      <th class="tabex" scope="col" width="375">Volume/Weight</th>
    </tr>
    <tr>
      <td class="tabex">acrylamide solution (30%)</td>
      <td class="tabex" width="210">13.3 ml</td>
    </tr>
    <tr>
      <td class="tabex">10 x TBE</td>
      <td class="tabex" width="210">4 ml</td>
    </tr>
    <tr>
      <td class="tabex">water</td>
      <td class="tabex" width="210">22.7 ml</td>
    </tr>
    <tr>
      <td class="tabex">total</td>
      <td class="tabex" width="210">40 ml</td>
    </tr>
  </tbody>
 </table>
 <br>
    
    
    <table class="experiment" width="400" border="1" style="margin-bottom:10px;">
    <caption align="left" style=" caption-side:top;">
       <strong> 6. Sequence information</strong>
      </caption>
  <tbody>
    <tr>
      <th class="tabex" scope="col" style="padding-left: 30px;padding-right: 30px;">Content</th>
      <th class="tabex" scope="col" style="padding-left: 30px;padding-right: 30px;">Sequence</th>
    </tr>
    <tr>
      <td  class="tabex" style="padding-left: 30px;padding-right: 30px;">Strand 1 </td>
      <td  class="tabex" style="border-bottom: 1 solid rgb(245,245,245);padding-left: 20px;padding-right: 20px;">GAGTCCGTACCATCACATATTCACTATCAA<br>CCTATCATTCATAAGTCACTCAGGCTGTACGGACTCGCGC </td>
    </tr>
    
    <tr>
      <td  class="tabex" style="padding-left: 20px;padding-right: 20px;">Strand 2</td>
      <td class="tabex" style="border-bottom: 1 solid rgb(245,245,245);padding-left: 20px;padding-right: 20px;">GACAGAGTTGAGCCTGAGTGACTTATGAAT<br>GATAGGTTGATAGTGAATATGTGATGCAACTCTGTCCGCG</td>
    </tr>
  </tbody>
</table>
</ul>
 <p class="figure" style="text-align:center; font-size:18px;">&nbsp;&nbsp; &nbsp;&nbsp; &nbsp;&nbsp;&nbsp;*【内容缺失】All DNA strands were synthesized and purified by Sangon Biotech. </p><br><br>
 <p style="margin-left:0px;"><img width="750" src="assets/images/experiment/6.png"></p>

 	 <p class="figure" style="text-align:center;"><strong>Figure 6. </strong>【图片缺失】</p><br><br>

</div>

<div id="float06" style="text-align:justify;">
    <h3 style="font-size:30px;font-weight: bolder;text-align: justify;">Methods</h3><br>

    <ul class="list-unstyled">
      <li style=" font-size: 24px;"><i class="fa fa-star"></i><strong> ---DNA phosphorylation</strong></li>
      <ol>
          <li style="list-style-type: none;">(a)---</li>
          <li style="list-style-type: none;">(b)---</li>
          <li style="list-style-type: none;">(c) ---</li>
         
      </ol><br>
      <li style=" font-size: 24px;"><i class="fa fa-star"></i> <strong>---DNA self connection</strong> </li>
      <ol>
          <li style="list-style-type: none;">System: 20uL</li>
          <li style="list-style-type: none;">P-ss DNA (40uM)     2.5uL</li>
<li style="list-style-type: none;">10× T4 buffer       2uL</li>
<li style="list-style-type: none;">H<sub>2</sub>O                14.5uL</li>
<li style="list-style-type: none;">T4 Dnl (5U/uL)       1uL</li>
<li style="list-style-type: none;">Set the temperature of the PCR instrument to 95 ℃ for 3 minutes, cool to 65 ℃/S at 0.1 ℃/s, and cool to 37 ℃ at 0.1 ℃/s for 3min. After that, Add T4 ligase, and overnight at 37 ℃ after enzyme addition (10h).</li>
      </ol><br>
       <li style=" font-size: 24px;"><i class="fa fa-star"></i> <strong>---Restriction enzyme digestion</strong></li>
      <ol>
          <li style="list-style-type: none;">Take 10ul of self connecting product and add:</li>
          <li style="list-style-type: none;">Exo1                   0.3ul</li>
          <li style="list-style-type: none;">Exo3                   0.5ul</li>
<li style="list-style-type: none;">Exo1 buffer             0.5ul</li>
<li style="list-style-type: none;">Exo3 buffer             0.5ul</li>
<li style="list-style-type: none;">Reaction for 4 hours.</li>
      </ol><br>
      <li style=" font-size: 24px;"><i class="fa fa-star"></i> <strong>---Hybridization of dumbbell structure</strong></li>
      <ol>
          <li style="list-style-type: none;">System: 10uL</li>
          <li style="list-style-type: none;">Self connecting product1        2uL</li>
          <li style="list-style-type: none;">Self connecting product2        2uL</li>
          <li style="list-style-type: none;">Mgcl2 (100mM)               1uL</li>
<li style="list-style-type: none;">Hepes buffer (Ph7.5 100mM)     1uL</li>
<li style="list-style-type: none;">H<sub>2</sub>O                         4uL</li>
<li style="list-style-type: none;">Hybridization time: 95 ℃ for 3 minutes, cooled to 37 ℃ at 0.1 ℃/s, overnight (10 hours).</li>
      </ol><br>
       <li style=" font-size: 24px;"><i class="fa fa-star"></i> <strong>---</strong></li>
      <ol>
          <li style="list-style-type: none;">---</li>
          <li style="list-style-type: none;">---</li>
          <li style="list-style-type: none;">---</li>
          <li style="list-style-type: none;">---</li>
          <li style="list-style-type: none;">---</li>
          <li style="list-style-type: none;">---</li>
      </ol><br>
      <li style=" font-size: 24px;"><i class="fa fa-star"></i> <strong>---</strong></li>
    </ul>
    </div>
    </section><!--//feature-video--> 
  <!-- ******FAQ Section****** --> 
    <section class="faq section has-bg-color" id="expertable" style="margin-top: -130px;margin-left: 100px;">
        <div class="container" id="exper">
<div class="row">
        <div class="col-md-8 col-sm-10 col-xs-12 col-md-offset-2 col-sm-offset-1 col-xs-offset-0">
                    <div class="panel">
                        <div class="panel-heading">
                            <h4 class="panel-title"><a data-parent="#accordion"
                            data-toggle="collapse" class="panel-toggle" href="#faq1"><i class="fa fa-plus-square"></i>Preparing input DNA</a></h4>
                        </div>
            
                        <div class="panel-collapse collapse" id="faq1">
                            <div class="panel-body">
                               <p> 1. Check the quality, quantity and formulation of the input DNA. Purity as measured using Nanodrop - OD 260/280 of 1.8 and OD 260/230 of 2.0-2.2; Input mass, as measured by Qubit - 1 µg.</p>
                               <p>2. Transfer 1-1.5 μg input DNA into a DNA LoBind tube and adjust the volume to 46 μl with nuclease-free water. Mix thoroughly by inversion and spin down briefly in a microfuge.</p>
                            </div>
                        </div>
                    </div><!--//panel-->
            
                    <div class="panel">
                        <div class="panel-heading">
                            <h4 class="panel-title"><a data-parent="#accordion"
                            data-toggle="collapse" class="panel-toggle" href="#faq2"><i class="fa fa-plus-square"></i>End-prep</a></h4>
                        </div>
            
                        <div class="panel-collapse collapse" id="faq2">
                            <div class="panel-body">
                                <p> 3. Transfer the 45 μl sample to a 1.5 ml DNA LoBind Eppendorf tube.Add 7 μl Ultra II End-prep reaction buffer; 3 μl Ultra II End-prep enzyme mix and 5 μl Nuclease-free water in the tube.</p>
                               <p>4. Transfer the sample to a 0.2 ml PCR tube, and incubate for 5 minutes at 20°C and 5 minutes at 65°C using the thermal cycler.</p>
                               <p>5. Transfer the sample to a 1.5 ml DNA LoBind Eppendorf tube.Add 60 µl of resuspended AMPure XP beads to the end-prep reaction and mix by pipetting. Incubate on a Hula mixer (rotator mixer) for 5 minutes at room temperature.</p>
                               <p>6. Prepare 500 μl of fresh 70% ethanol in nuclease-free water.</p>
                               <p>7. Spin down the sample and pellet on a magnet. Keep the tube on the magnet, and pipette off the supernatant.</p>
                               <p>8. Keep on magnet, wash beads with 200 µl of freshly prepared 70% ethanol without disturbing the pellet. Remove the 70% ethanol using a pipette and discard. Repeat.</p>
                               <p>9. Spin down and place the tube back on the magnet. Pipette off any residual ethanol. Allow to dry for ~30 seconds.</p>
                               <p>10. Remove the tube from the magnetic rack and resuspend pellet in 31 µl nuclease-free water. Incubate for 2 minutes at room temperature. Pellet beads on magnet until the eluate is clear and colourless.</p>
                               <p>11. Remove and retain 31 µl of eluate into a clean 1.5 ml Eppendorf DNA LoBind tube. Quantify 1 µl of end-prepped DNA using a Qubit fluorometer(nanodrop) - recovery aim > 700 ng.</p>
                            </div>
                        </div>
                    </div><!--//panel-->
            
                    <div class="panel">
                        <div class="panel-heading">
                            <h4 class="panel-title"><a data-parent="#accordion"
                            data-toggle="collapse" class="panel-toggle" href="#faq3"><i class="fa fa-plus-square"></i>Adapter ligation</a></h4>
                        </div>
            
                        <div class="panel-collapse collapse" id="faq3">
                            <div class="panel-body">
                                <p> 12. Thaw and prepare Adapter Bead Binding Buffer (ABB) and Elution Buffer (ELB) at room temperature. Thaw and prepare Adapter Mix 1D (AMX1D) ; Running Buffer with Fuel Mix (RBF) and Blunt/TA Ligation Master Mix on ice. Check the contents of each tube are clear of any precipitate and are thoroughly mixed before setting up the reaction.</p>
                               <p>13. Taking the end-prepped DNA, perform adapter ligation by mixing 30 µl End-prepped DNA 
20 µl Adapter Mix and 50 µl Blunt/TA Ligation Master Mix, mixing by flicking the tube between each sequential addition. Mix gently by flicking the tube, and spin down. Incubate the reaction for 10 minutes at room temperature.</p>

                            </div>
                        </div>
                    </div><!--//panel-->
                    
                    <div class="panel">
                        <div class="panel-heading">
                            <h4 class="panel-title"><a data-parent="#accordion"
                            data-toggle="collapse" class="panel-toggle" href="#faq4"><i class="fa fa-plus-square"></i>AMPure XP bead binding</a></h4>
                        </div>
            
                         <div class="panel-collapse collapse" id="faq4">
                            <div class="panel-body">
                                <p> 14. Prepare the AMPure XP beads for use, resuspend by vortexing.</p>
                               <p>15. Add 40 µl of resuspended AMPure XP beads to the adapter ligation reaction from the previous step and mix by pipetting. Incubate on a Hula mixer (rotator mixer) for 5 minutes at room temperature. Place on magnetic rack, allow beads to pellet and pipette off supernatant.</p>
                               <p>16. Add 140 μl of Adapter Bead Binding Buffer (ABB) to the beads. Close the tube lid, and resuspend the beads by flicking the tube. Return the tube to the magnetic rack, allow beads to pellet and pipette off the supernatant. Repeat.</p>
                               <p>17. Remove the tube from the magnetic rack and resuspend pellet in 15 µl Elution Buffer. Incubate for 10 minutes at room temperature.Pellet beads on magnet until the eluate is clear and colourless.</p>
                               <p>18. Remove and retain 15 µl of eluate into a clean 1.5 ml Eppendorf DNA LoBind tube.</p>
                               <p>19. Quantify 1 µl of adapter ligated DNA using a Qubit fluorometer(nanodrop) - recovery aim ~430 ng.</p>
                            </div>
                        </div>
                    </div><!--//panel--> 
                    
                   
                    
                    <div class="panel">
                        <div class="panel-heading">
                            <h4 class="panel-title"><a data-parent="#accordion"
                            data-toggle="collapse" class="panel-toggle" href="#faq6"><i class="fa fa-plus-square"></i>Priming and loading the SpotON flow cell</a></h4>
                        </div>
            
                        <div class="panel-collapse collapse" id="faq6">
                            <div class="panel-body">
                               <p> 20. Flip back the MinION lid and slide the priming port cover clockwise so that the priming port is visible.</p>
                               <p>21. After opening the priming port, check for small bubble under the cover. Draw back a small volume to remove any bubble.</p>
                               <p>22. Prepare the flow cell priming mix(576 µl RBF and 624 µl Nuclease-free water) in a clean 1.5 ml Eppendorf DNA LoBind tube.</p>
                               <p>23. Load 800 µl of the priming mix into the flow cell via the priming port, avoiding the introduction of air bubbles. Wait for 5 minutes.Thoroughly mix the contents of the RBF and LLB tubes by pipetting.</p>
                               <p>24. Mix 35.0 µl RBF;25.5 µl LLB; 2.5 µl Nuclease-free water and 12 µl DNA library to prepare the library for loading. </p>
                               <p>25. Gently lift the SpotON sample port cover to make the SpotON sample port accessible and load 200 µl of the priming mix into the flow cell via the priming port to complete the flow cell priming of air bubbles.</p>
                               <p>26. Mix the prepared library gently by pipetting up and down just prior to loading.</p>
                               <p>27. Add 75 μl of sample to the flow cell via the SpotON sample port in a dropwise fashion. Ensure each drop flows into the port before adding the next. </p>
                               <p>28. Gently replace the SpotON sample port cover, making sure the bung enters the SpotON port, close the priming port and replace the MinION lid.
</p>
                            </div>
                        </div>
                    </div><!--//panel-->

                     <div class="panel">
                        <div class="panel-heading">
                            <h4 class="panel-title"><a data-parent="#accordion"
                            data-toggle="collapse" class="panel-toggle" href="#faq5"><i class="fa fa-plus-square"></i>Starting a sequencing run</a></h4>
                        </div>
            
                        <div class="panel-collapse collapse" id="faq5">
                            <div class="panel-body">
                                <p> 29. Use MinKNOW to start sequencing and data analysis.</p>
                               <p>30. Close down MinKNOW and the Desktop Agent.</p>
                               <p>31. Prepare the flow cell for re-use.</p>
                            </div>
                        </div>
                    </div><!--//panel-->
                    <p>*MinION is purchased  from Oxford Nanopore Technologies.
</p>
                                          
      </div>
          </div><!--//row-->
      </div>

            <div class="contact-lead text-center"> </div>
      </div><!--//container-->        
    </section><!--//faq-->
</div>
    <!-- ******FOOTER****** --> 
 <footer class="footer">
        <P><strong>@2018TeamSeason</strong></P>
          <p><strong>Email: biomod2019@163.com</strong></p>
          <p><strong><a  href="https://drnatech.wixsite.com/mysite"> LAB </a>&nbsp;&nbsp;&nbsp;<a href="http://biomod.net/">BIOMOD</a></strong></p>
    </footer><!--//footer-->
    
    <!-- Javascript -->   
       <script type="text/javascript" src="assets/js/jquery.min.js"></script>
   <script src="assets/js/jquery-1.7.1.min.js"></script>
   <script type="text/javascript">
	$(".list_dt").on("click",function () {//给上层项目添加点击时的函数
		$('.list_dd').stop();//停止下层标目动画
		$(this).siblings("dt").removeAttr("id");//找到所有的上层项目并去掉它的id
		if($(this).attr("id")=="open"){//获取id的值若与open相等
			$(this).removeAttr("id").siblings("dd").slideUp();//假如上层项目id是open，即该下拉菜单是展开的，就会移除该id并且以滑动的形式隐藏下级项目
		}else{
			$(this).attr("id","open").next().slideDown().siblings("dd").slideUp();
		}//总体来说就是原先是打开的点击就关上，原先隐藏的点击就下拉
	});





    $(function(){
       var AllHet = $(window).height()//窗口宽度
       var mainHet= $('.floatCtro').height();//导航栏宽度
       var fixedTop = (AllHet - mainHet)/2//（窗口宽度-导航栏宽度）/2
     //  $('div.floatCtro').css({top:fixedTop+'px'}); 
       $('div.floatCtro li').click(function(){//导航栏的点击功能
            var ind = $('div.floatCtro li').index(this)+1;//导航栏所在索引+1，即当前所在导航栏位置
            var topVal = $('#float0'+ind).offset().top-95;//当前导航栏所对应区域距离浏览器上边缘的距离
            $('body,html').animate({scrollTop:topVal},500)//可见区域为当前导航栏对应区块距离上边缘的距离处开始，动画速度为1000秒
        })
        $('div.floatCtro a').click(function(){
            $('body,html').animate({scrollTop:0},1000)//一旦点击之后可见区域即为导航栏当前区块开始的位置，动画时间为000秒
            })
       $(window).scroll(scrolls)//scroll为滚动事件，结合具体函数scrolls使用
       scrolls()
       function scrolls(){
           var f1,f2,f3,f4,f5,f6,bck;
           var fixRight = $('div.floatCtro li');//导航栏的p
           var blackTop = $('div.floatCtro a')//导航栏返回顶部的链接
           var sTop = $(window).scrollTop();//获取滚动条距离窗口上边缘的位置
           fl = $('#float01').offset().top;
           f2 = $('#float02').offset().top;
           f3 = $('#float03').offset().top;
           f4 = $('#float04').offset().top;
           f5 = $('#float05').offset().top;
           f6 = $('#float06').offset().top;
           //分别为不同区块距离窗口顶部的距离
           
            
           if(sTop<=f2-100){//若滚动条的窗口小于第二区块距离窗口上边缘的距离-100
               blackTop.fadeOut(100).css('display','none')
               }//关于返回顶部的一个淡出效果
           else{
               blackTop.fadeIn(100).css('display','block')
               }//返回顶部的一个淡入效果
           
           if(sTop>=fl){
               fixRight.eq(0).addClass('cur').siblings().removeClass('cur');
               }//若滚动条距离窗口距离大于区块一下部与窗口距离，跳转到标目2
           if(sTop>=f2-100){
               fixRight.eq(1).addClass('cur').siblings().removeClass('cur');
               }
           if(sTop>=f3-100){
               fixRight.eq(2).addClass('cur').siblings().removeClass('cur');
               }
           if(sTop>=f4-100){
               fixRight.eq(3).addClass('cur').siblings().removeClass('cur');
               }
            if(sTop>=f5-100){
               fixRight.eq(4).addClass('cur').siblings().removeClass('cur');
               }
         if(sTop>=f6-100){
               fixRight.eq(5).addClass('cur').siblings().removeClass('cur');
               }
        
        
         }//导航栏移动的方法
       })




	//myself
	$('dt.list_dt a').click(function(){//导航栏的点击功能
            var ind = $('dt.list_dt a').index(this)+1;//导航栏所在索引+1，即当前所在导航栏位置
            var topVall = $('#same0'+ind).offset().top-95;//当前导航栏所对应区域距离浏览器上边缘的距离
            $('body,html').animate({scrollTop:topVall},500)//可见区域为当前导航栏对应区块距离上边缘的距离处开始，动画速度为1000秒
        })
	//jieshu
	</script>
    


           
    <script type="text/javascript" src="assets/plugins/jquery-1.12.3.min.js"></script>
    <script type="text/javascript" src="assets/plugins/bootstrap/js/bootstrap.min.js"></script> 
    <script type="text/javascript" src="assets/plugins/bootstrap-hover-dropdown.min.js"></script>
    <script type="text/javascript" src="assets/plugins/back-to-top.js"></script>
    <script type="text/javascript" src="assets/plugins/jquery-placeholder/jquery.placeholder.js"></script>
    <script type="text/javascript" src="assets/plugins/FitVids/jquery.fitvids.js"></script>  
    <script type="text/javascript" src="assets/plugins/flexslider/jquery.flexslider-min.js"></script> 
    <script type="text/javascript" src="assets/js/main.js"></script>
    <script src="assets/js/loading.js"></script>
     <!--为了浏览器兼容性jqeury版本推荐使用1.12.0，其他版本会出现不支持某些浏览器-->

            
</body>
</html> 


